src/client/components/series-menu/series-menu.tsx

```diff
@@ -33,7 +34,8 @@ interface SeriesMenuProps {
   openOn: Element;
   containerStage: Stage;
   onClose: Fn;
-  series: Series;
+  // TODO: fix
+  series: any;
```

Comment:
`SeriesMenu` will be polymorphic and will render differently for `MeasureSeries` and `ExpressionSeries`

Reply:
So put this meaningful comment instead of general "fix" in the code :)